perception/navigator_lidar_oa/src/ConnectedComponents.h

```diff
@@ -182,10 +185,9 @@ std::vector< std::vector<int> > ConnectedComponents(OccupancyGrid &ogrid, std::v
 	}
 
 	//Re-organize obstacles, how many connections to count as an obstacle?
-	int newId = 0;
 	objects.clear();
 	for (auto ii : mapObjects)  {
-		if (ii.second.strikesPersist.size() >= 10) {
```

Comment:
This might not be needed for this PR, but eventually magic numbers like this should be setup as rosparams, so they can easily be changed /viewed at runtime. This is as easy as:
```
 nh.param<int>("strikes_persist",STRIKE_PERSIST_VARIABLE, 10);
```
Where the string is the key we set the param with, the middle term is the variable to set by reference, and the third is a default. The template can be int,double,string etc.